moz-webgpu-cts/src/process_reports.rs

```diff
@@ -136,6 +137,10 @@ fn reconcile<Out>(
                     Some(rep) => meta | rep,
                     None => meta,
                 },
+                ReportProcessingPreset::SetNewOutcomes => |meta, rep| match rep {
+                    Some(rep) => rep,
+                    None => meta,
+                },
```

Comment:
thought: At first glance, this appears to be different from `ResetAll` WRT fallback (i.e., `Set` falls back to `meta`, rather than `ResetAll`'s `Default::default()`). But `ResetAll` actually doesn't ever run into the fallback case, since it currently deletes missing entries and skips reconciliation entirely. Huh...I hadn't considered that that wasn't exercised before. 😅

I think this is indicative of presets being somewhat leaky in this new dimension. We might want to change `ResetAll` to match this function body instead. 🤔